src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const liElements = document.querySelectorAll('li');
+
+for (const li of liElements) {
+  if (li.querySelector('ul')) {
+    const textNode = li.childNodes[0];
+    const correctText = textNode.textContent.trim();
+    const span = document.createElement('SPAN');
+
+    span.textContent = correctText;
+
+    li.replaceChild(span, textNode);
+  }
+}
+
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    if (e.target.nextSibling.style.display === 'none') {
```

Comment:
The condition `e.target.nextSibling.style.display === 'none'` assumes that `nextSibling` is an element with a `style` property. However, `nextSibling` could be a text node or `null`, leading to a runtime error. Consider checking if `nextSibling` is an element before accessing its `style` property.